packages/CopyInput/src/CopyInput.js

```diff
@@ -46,48 +47,65 @@ function CopyInput(props) {
 
   return (
     <sc.CopyInput data-pka-anchor="copy-input" {...moreProps}>
-      <Input ref={inputRef} defaultValue={defaultValue} {...extendedInputProps} />
-      <div ref={buttonRef}>
+      <Input ref={inputRef} defaultValue={value} isReadOnly={isReadOnly} {...extendedInputProps} />
+      <div ref={buttonRef} data-pka-anchor="copy-input.button">
         <Button.Icon
           kind="primary"
           onClick={handleButtonClick}
-          onMouseEnter={() => setIsHoverTooltipOpen(true)}
-          onMouseLeave={() => setIsHoverTooltipOpen(false)}
+          onMouseOver={() => setIsHoverTooltipOpen(true)}
+          onMouseOut={() => setIsHoverTooltipOpen(false)}
+          onFocus={() => setIsHoverTooltipOpen(true)}
+          onBlur={() => setIsHoverTooltipOpen(false)}
         >
           <CopyIcon />
         </Button.Icon>
       </div>
       <Popover
+        a11yText={isClickedTooltipOpen ? I18n.t("copyInput.clicked_tooltip") : I18n.t("copyInput.hover_tooltip")}
         align="bottom"
+        getPositioningElement={() => buttonRef.current}
         isDark
+        isEager
         isOpen={isHoverTooltipOpen && !isClickedTooltipOpen}
-        getPositioningElement={() => buttonRef.current}
       >
         <Popover.Content>
           <Popover.Card>{I18n.t("copyInput.hover_tooltip")}</Popover.Card>
         </Popover.Content>
         <Popover.Tip />
       </Popover>
-      <Popover align="bottom" isOpen={isClickedTooltipOpen} getPositioningElement={() => buttonRef.current}>
+      <Popover
+        align="bottom"
+        getPositioningElement={() => buttonRef.current}
+        isOpen={isClickedTooltipOpen}
+        shouldKeepFocus
+      >
         <Popover.Content>
           <Popover.Card>{I18n.t("copyInput.clicked_tooltip")}</Popover.Card>
         </Popover.Content>
         <Popover.Tip />
       </Popover>
+      {isClickedTooltipOpen && (
+        <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} aria-hidden>
+          Copied
+        </Toast>
+      )}
     </sc.CopyInput>
   );
 }
 
 const propTypes = {
   /** Used for CopyInput.Input */
   children: PropTypes.node,
-  /** Default value for the input. */
-  defaultValue: PropTypes.string,
+  /** Is the input read-only. */
+  isReadOnly: PropTypes.bool,
```

Comment:
This _could_ simply be added to the `<CopyInput.Input>` without being an explicit part of the `<CopyInput>` API.  Or did you have a reason you thought it would be good to have here?

Reply:
Hmm i guess it might as well be just be a prop for `CopyInput.Input`, since the user probably wants it to be a controlled component if the input is editable. But since `isReadOnly` should default `false`, it's a bit weird to have it as a `CopyInput.Input` prop, since the user would expecting the normal `Input` default props. If we have `isReadOnly` props is at the root level component, the API is more clear.

Reply:
Fair point. I'll let you make the decision – it's your component 😄

Reply:
Alright, i'll leave it as it is. Users who just want an uncontrolled editable CopyInput can rejoice since they won't need to give the component any children